bundle/manifests/sailoperator.clusterserviceversion.yaml

```diff
@@ -472,6 +472,12 @@ spec:
                 - horizontalpodautoscalers
               verbs:
                 - '*'
+            - apiGroups:
+                - discovery.k8s.io
+              resources:
+                - endpointslices
+              verbs:
+                - '*'
```

Comment:
Don't we want to limit this to just get, list and watch?

Reply:
Good call, that will be sufficient for now